File: fsharp-giraffe-server/OpenAPI/src/model/ResponseRoutesResult.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open OpenAPI.Model.ResponseRoutesLocation

module ResponseRoutesResult = 

  //#region ResponseRoutesResult


  type ResponseRoutesResult = {
    SearchId : string;
    Locations : ResponseRoutesLocation[];
    Unreachable : string[];
  }
  //#endregion
  

File: scala-play-server/app/model/ResponseRoutesResult.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ResponseRoutesResult.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2020-07-14T11:29:17.777Z[Etc/UTC]")
case class ResponseRoutesResult(
  searchId: String,
  locations: List[ResponseRoutesLocation],
  unreachable: List[String]
)

object ResponseRoutesResult {
  implicit lazy val responseRoutesResultJsonFormat: Format[ResponseRoutesResult] = Json.format[ResponseRoutesResult]
}

